### DISCUSSION TITLE
фарминг

### DISCUSSION THREAD

**Discussion by igor19801** [Chia-Network/chia-blockchain#8597] [Support]
установил версию Блокчейн Chia 1.2.7 участки загрузились на 100%, фарминг не загружается, что делать?

**Comment by loppefaaret** [comment]
if you could provide additional information, such as do you see any errors ? what leads you to believe that you are not farming ?

**Reply by igor19801** [reply]
перезагрузил систему виндус. сейчас идет синхронизация.

**Comment by BrandtH22** [comment]
@igor19801 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
